examples/next-prisma-starter/.eslintrc

```diff
@@ -7,6 +7,7 @@
     "plugin:prettier/recommended"
   ],
   "parserOptions": {
+    "project": "tsconfig.json",
```

Comment:
This doesn't look related to this PR? We only need to change config at the root level of the project

```suggestion
```